root/static/scripts/guess-case/MB/GuessCase/Main.js

```diff
@@ -1,7 +1,6 @@
 /*
    This file is part of MusicBrainz, the open internet music database.
-   Copyright (c) 2005 Stefan Kestenholz (keschte)
-   Copyright (C) 2010 MetaBrainz Foundation
+   Copyright (C) 2016 MetaBrainz Foundation
```

Comment:
Definitely not “2010–2016”, because that would mean that a copyrightable change has been made in each of the years 2010, 2011, 2012, 2013, 2014, 2015, and 2016.

In this case, I agree that the change is too minor to justify extending the copyright statement in whichever way.